starter-code/apple-pie-starter-code/css/main.css

```diff
@@ -0,0 +1,65 @@
+/*Some necessary adjustments*/
+h1, body, html, main {
+  margin: 0;
+}
+/*Import the font and set the default size*/
+html {
+  font-family: 'Open Sans', sans-serif;
+  font-size: 18px
+}
+
+hr{
+  max-width: 80%;
+}
+
+b {
+  padding-left: 20%;
+}
+
+.headerimage {
+  background-image: url("../images/apple-pie.jpg");
+  background-position: left top;
+  background-repeat: none;
+  background-size: cover;
```

Comment:
If you would like to fit the whole photo in div (since only half of it is visible now), try adding something like:
```
background: url("../images/apple-pie.jpg") no-repeat center center;
background-size: 100%;
```